doc/guidance.md

```diff
@@ -14,9 +14,22 @@ This is about how to use Doxygen to maintain the API documents. There are three
 	- on mac, run "brew install graphviz"
 - goto the root directory of "amazon-freertos" (this will be referenced as $ROOT_AFR_DIR)
 	- cd $ROOT_AFR_DIR
-- run Doxygen command to generate the documents 
-	- doxygen doc/config/main
+- create a tag folder to store doxygen generated tags
+	- mkdir $ROOT_AFR_DIR/doc/tag
+- run Doxygen commands to generate the documents
+	- doxygen doc/config/linear_containers
+	- doxygen doc/config/static_memory
+	- doxygen doc/config/logging
 	- doxygen doc/config/secure_sockets
+	- doxygen doc/config/wifi
+	- doxygen doc/config/atomic
+	- doxygen doc/config/taskpool
+	- doxygen doc/config/platform
+	- doxygen doc/config/mqtt
```

Comment:
missing posix

Reply:
The config file for posix appears to be missing